requirements-dev.txt

```diff
@@ -0,0 +1,2 @@
+Kodistubs
+DBussy
```

Comment:
what purpose has this file ?

Reply:
Those are minimal requirements for a development environment: https://pip.pypa.io/en/stable/user_guide/#requirements-files
If someone wants to develop the addon using an IDE like PyCharm or VScode those are the dependencies they need to install to have proper code introspection.